Demo/src/main/java/com/braintreepayments/demo/DemoActivity.java

```diff
@@ -80,13 +80,15 @@ public BraintreeClient getBraintreeClient() {
         return braintreeClient;
     }
 
+    // TODO: Remove this method completely when all payment methods have been migrated to
```

Comment:
We could delete this code since we're merging into `next-minor-version`.

Reply:
👍